pkg/controller/podautoscaler/replica_calculator_test.go

```diff
@@ -568,6 +568,38 @@ func TestReplicaCalcScaleUpUnreadyNoScale(t *testing.T) {
 	tc.runTest(t)
 }
 
+func TestExternalPerPodMetricReplicaOverflow(t *testing.T) {
+	tc := replicaCalcTestCase{
+		currentReplicas:  1,
+		expectedReplicas: math.MaxInt32,
+		metric: &metricInfo{
+			name:              "qps",
+			levels:            []int64{math.MaxInt64},
```

Comment:
I don't quite understand the purpose of this test case. Does our test case test the effect of int overflow?